static-site/content/featured-analyses.yaml

```diff
@@ -78,3 +78,8 @@
   description: Rabies virus evolution and spread
   img: rabies_tree.png
   url: /rabies
+
+- name: Human Metapneumovirus
```

Comment:
Maybe abbreviate name to 'HMPV', the current display cuts off the description

![Screenshot 2024-12-18 at 10 57 38 AM](https://github.com/user-attachments/assets/df569837-0baa-4f90-af99-377de2e568f0)

Reply:
good call, updated and force pushed.